### DISCUSSION TITLE
rsl_rl reproducability

### DISCUSSION THREAD

**Discussion by SourabhPrasad** [isaac-sim/IsaacLab#2123] [Q&A]
Hi,
I'm working on training a quadruped robot using a modified version of rsl_rl that includes an auxiliary MLP network. 
I've encountered a problem with reproducibility: even when seed is explicitly set while executing the training script. 
Do I need to manually set the seed anywhere else in addition to DirectRL's `torch_utils.set_seed()`

![Screenshot from 2025-03-20 17-30-53](https://github.com/user-attachments/assets/01a2ef3a-b6ef-42ad-accc-16947dc01f56)

**Comment by RandomOakForest** [comment]
Thanks for posting this. Using `--seed` when launching `train.py` should allow you to reproduce a model.

**Reply by SourabhPrasad** [reply]
Hey @RandomOakForest,
Thanks for the reply. The `--seed=42` was set while executing the training script.
